src/pages/Home.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const Home = () => {
+  const titleStyle = {
+    color: 'black',
+    fontSize: '2.3rem',
+  };
+
+  const paragraphStyle = {
+    color: 'black',
+    fontSize: '1.3rem',
+  };
+
+  return (
+    <div>
+      <h2 style={titleStyle}>Welcome to Math-Magicians!</h2>
+      <p style={paragraphStyle}>
+        &quot;Math magicians&quot; is a website for all fans of mathematics.
+        <span> </span>
+        It is a Single Page App (SPA) that allows users to:
+        <ul>
+          <li>Make simple calculations.</li>
+          <li>Read a random math-related quote.</li>
```

Comment:
Please ensure that the general layout matches these wireframes.

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/166108118-d08698d2-dc6f-4090-995f-982c52e2f474.png) | ![image](https://user-images.githubusercontent.com/10332499/166108105-503a44c8-9824-4259-9a7b-54d976065468.png)

Reply:
@micheaol  Thanks! I already knew of this problem. I would like to ask you if you can help me in your spare time to figure this out since I am out of ideas.

Reply:
@AdrianCSM 
It would be my pleasure. I will let you know when I am free.